test/Full.t.sol

```diff
@@ -106,130 +42,83 @@ contract FullTest is Test {
-        assertFalse(controller.hasRole(controller.EXECUTOR_ROLE(), EMERGENCY_MS));
-        assertFalse(controller.hasRole(controller.EXECUTOR_ROLE(), SCHEDULED_UPDATE_MS));
-        assertTrue(controller.hasRole(controller.EXECUTOR_ROLE(), EMERGENCY_UPDATE_MS));
-        assertTrue(controller.hasRole(controller.EXECUTOR_ROLE(), MAINTENANCE_MS));
-        assertTrue(controller.hasRole(controller.EXECUTOR_ROLE(), address(timelock)));
-
-        // Cancellers
-        assertFalse(controller.hasRole(controller.CANCELLER_ROLE(), RESUME_MS));
-        assertFalse(controller.hasRole(controller.CANCELLER_ROLE(), EMERGENCY_MS));
-        assertFalse(controller.hasRole(controller.CANCELLER_ROLE(), MAINTENANCE_MS));
-        assertFalse(controller.hasRole(controller.CANCELLER_ROLE(), SCHEDULED_UPDATE_MS));
-        assertTrue(controller.hasRole(controller.CANCELLER_ROLE(), address(timelock)));
-        assertTrue(controller.hasRole(controller.CANCELLER_ROLE(), EMERGENCY_UPDATE_MS));
-    }
 
-    function testTimelockRoles() public {
-        // Admin
-        assertFalse(timelock.hasRole(timelock.TIMELOCK_ADMIN_ROLE(), address(this)));
-        assertFalse(timelock.hasRole(timelock.TIMELOCK_ADMIN_ROLE(), MAINTENANCE_MS));
-        assertFalse(timelock.hasRole(timelock.TIMELOCK_ADMIN_ROLE(), SCHEDULED_UPDATE_MS));
-        assertFalse(timelock.hasRole(timelock.TIMELOCK_ADMIN_ROLE(), RESUME_MS));
-        assertFalse(timelock.hasRole(timelock.TIMELOCK_ADMIN_ROLE(), EMERGENCY_MS));
-        assertTrue(timelock.hasRole(timelock.TIMELOCK_ADMIN_ROLE(), EMERGENCY_UPDATE_MS));
-
-        // Proposers
-        assertFalse(timelock.hasRole(timelock.PROPOSER_ROLE(), MAINTENANCE_MS));
-        assertFalse(timelock.hasRole(timelock.PROPOSER_ROLE(), RESUME_MS));
-        assertFalse(timelock.hasRole(timelock.PROPOSER_ROLE(), EMERGENCY_MS));
-        assertTrue(timelock.hasRole(timelock.PROPOSER_ROLE(), EMERGENCY_UPDATE_MS));
-        assertTrue(timelock.hasRole(timelock.PROPOSER_ROLE(), SCHEDULED_UPDATE_MS));
-
-        // Executors
-        assertFalse(timelock.hasRole(timelock.EXECUTOR_ROLE(), RESUME_MS));
-        assertFalse(timelock.hasRole(timelock.EXECUTOR_ROLE(), EMERGENCY_MS));
-        assertTrue(timelock.hasRole(timelock.EXECUTOR_ROLE(), SCHEDULED_UPDATE_MS));
-        assertTrue(timelock.hasRole(timelock.EXECUTOR_ROLE(), EMERGENCY_UPDATE_MS));
-        assertTrue(timelock.hasRole(timelock.EXECUTOR_ROLE(), MAINTENANCE_MS));
-
-        // Cancellers
-        assertFalse(timelock.hasRole(timelock.CANCELLER_ROLE(), RESUME_MS));
-        assertFalse(timelock.hasRole(timelock.CANCELLER_ROLE(), EMERGENCY_MS));
-        assertFalse(timelock.hasRole(timelock.CANCELLER_ROLE(), MAINTENANCE_MS));
-        assertTrue(timelock.hasRole(timelock.CANCELLER_ROLE(), SCHEDULED_UPDATE_MS));
-        assertTrue(timelock.hasRole(timelock.CANCELLER_ROLE(), EMERGENCY_UPDATE_MS));
+        // TODO: current multisig must pass over ownership of contract to controller
```

Comment:
This TODO seems to be obsolete

Reply:
The TODO is there just as a heads-up that the proper transfer of ownerships should be done with the multisig.